app/(site)/events/page.tsx

```diff
@@ -1,7 +1,154 @@
+
+  },
+  {
+    name: "SPELL BEE",
+    image: "/images/spell_bee.jpg",
+    description: "Classic Spelling-Bee competition for word geeks of all ages.",
+
+  },
+  {
+    name: "STORY BOX",
+    image: "/images/STORY_BOX.jpg",
+    description: "An event that tests the enduring power of storytelling.",
+
+  },
+  {
+    name: "NEWSROOM",
+    image: "/images/newsroom.jpg",
+    description: "An overall fun event that tests grammatical precision and oratory skills followed by a panel discussion.",
+  },
+  {
+    name: "VAAK-YUDDH",
+    image: "/images/vaak_yuddh.jpg",
+    description: "Comprises of two rounds, first : instant poetry writing on certain topic followed by a poetic version of debate between the respective teams.",
+  },
+  {
+    name: "RADIOACTIVE",
+    image: "/images/radioactive.jpg",
+    description: "Participants are expected to connect the two songs with their impeccable enunciation and witty punchlines in a stipulated time.",
+  },
+  {
+    name: "MIC DROP",
+    image: "/images/mic_drop.jpg",
+    description: "A platform to flaunt your pop culture and drama obsessions by answering questions and debating characters.",
+  },
+  {
+    name: "GUEST SESSION",
+    image: "/images/guest_session.jpg",
+    description: "Guest session aims to share amusing anecdotes and valuable insights from an eminent literary personality.",
+  },
+  /*{
+    name: "LITERARY TALK",
+    image: "/images/literary_talk.jpg",
+    description: "A thorough discussion on a specific topic of literature where diverse facets of written and spoken words take new forms upholding the nuances of literature.",
+  },*/
+  {
+    name: "NIGHTS",
+    image: "/images/nights.jpg",
+    description: "Enthralling special performances by various renowned artists and performers are a special aspect of our literary fest.",
+  },
+  {
+    name: "KAVI SAMMELAN",
+    image: "/images/kavi_sammelan_rostra.jpg",
+    description: "A gathering filled with captivating realms of poetries that instil a plethora of emotions in the audience, leaving them awestruck and amazed.",
+  },
+];
+
+const litnightEvents = [
+  {
+    name: "THESPIAN",
+    image: "/images/thespian.jpg",
+    description: "Thespian is the rhetoric of theatre where the contestants impersonate their favorite characters and engage in a contentious debate of contrasting opinions and immaculate wits.",
+  },
+  {
+    name: "KAVI SAMMELAN",
+    image: "/images/kavi_sammelan.jpg",
+    description: "A gathering filled with captivating realms of poetries that instil a plethora of emotions in the audience, leaving them awestruck and amazed.",
+  },
+  {
+    name: "SYMPOSIUM",
+    image: "/images/symposium.jpg",
+    description: "It is a contest that pledges an opportunity for literature enthusiasts to share their soulful stories, heartfelt poetries and writeups on fervent narratives.",
+  },
+];
```

Comment:
instead of adding these configs here, upload these files on supabase in a separate events table and call from there. will be a better practise for later changes